app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -376,7 +376,7 @@ class MainActivity :
                             VR.id.navigation_profile -> {
                                 if (settings.getEndOfYearModalHasBeenShown()) {
                                     binding.bottomNavigation.removeBadge(VR.id.navigation_profile)
-                                    settings.setEndOfYearShowBadge2022(false)
+                                    settings.setEndOfYearShowBadge2023(false)
                                 }
```

Comment:
It seems that if the screen opens to the profile view, the badge is shown but tapping on the badge doesn't remove it. Instead you have to navigate away and then come back. Not a big deal or a blocker, but a mild nuisance.


https://github.com/Automattic/pocket-casts-android/assets/4656348/7e8af749-5b26-4490-9bbf-e89ffb0f60a2

Reply:
Fixed in https://github.com/Automattic/pocket-casts-android/pull/1503/commits/2c84eceeb87f716128972250a9e55cd895898e

https://github.com/Automattic/pocket-casts-android/assets/1405144/66cc90a2-621b-4d53-9c7e-dc52fe05a41c